oteapi_dlite/strategies/function.py

```diff
@@ -15,7 +14,7 @@
 from pydantic.dataclasses import dataclass
 
 from oteapi_dlite.models import DLiteSessionUpdate
-from oteapi_dlite.utils import get_driver, init_session
+from oteapi_dlite.utils import get_collection, get_driver
```

Comment:
This is the same change as suggested in #87?

Reply:
Yes, PR #87 builds on top on this PR